main.go

```diff
@@ -168,6 +174,11 @@ func main() {
 			"Insecure values: "+strings.Join(tlsCipherInsecureValues, ", ")+".")
 	flag.IntVar(&controllerConcurrency, "controller-concurrency", 0,
 		"Number of CRs of each type to process simultaneously")
+
+	flag.StringVar(&leaseDuration, "lease-duration", os.Getenv("LEASE_DURATION"), "Golang duration, e.g. '26s'")
```

Comment:
Help strings could be something like `Leader election lease duration with units, e.g. "15s"`

Ideally we should use the [defaults](https://github.com/kubernetes-sigs/controller-runtime/blob/479b723944e34ae42c9911fe01228ff34eb5ca81/pkg/manager/manager.go#L210-L221) (which are 15s, 10s, and 2s, respectively) as the examples.